Formula/das_tool.rb

```diff
@@ -0,0 +1,42 @@
+class DasTool < Formula
+  # cite Sieber_2018: "https://doi.org/10.1038/s41564-018-0171-1"
+  desc "Genomic binning refiner"
+  homepage "https://github.com/cmks/DAS_Tool"
+  url "https://github.com/cmks/DAS_Tool/archive/1.1.1.tar.gz"
+  sha256 "2a55f67b5331251d8fd5adea867cc341363fbf7fa7ed5c3ce9c7679d8039f03a"
+  head "https://github.com/cmks/DAS_Tool.git"
+
+  if OS.mac?
+    depends_on "coreutils"
```

Comment:
Could you open a bug report upstream with `DAS_Tool` that it doesn't work with the `/usr/bin/split` provided by macOS? This dependency is fine in the mean time.

Reply:
Done.

https://github.com/cmks/DAS_Tool/issues/36

Also, I've changed the patch in the formula to the one I proposed in the Issue. This then uses BSD-compatible split flags.

Reply:
Thanks!